imports/client/components/Puzzle.tsx

```diff
@@ -68,8 +67,9 @@ const StyledButton = styled(Button)`
   }
 `;
 
-const PuzzleTitleColumn = styled(Breakable)`
+const PuzzleTitleColumn = styled(PuzzleColumn)`
   flex: 4;
+  overflow-wrap: word;
```

Comment:
```suggestion
  overflow-wrap: break-word;
```
I think this is just a typo. With that change, I believe it works. Thanks for catching this.

Reply:
Ah, yeah, I misread `Breakable.tsx`.  Thanks for catching my wrong-fix!